src/StoreKeeper/WooCommerce/B2C/Models/AttributeOptionModel.php

```diff
@@ -24,12 +26,20 @@ public static function getFieldsWithRequired(): array
         ];
     }
 
+    /**
+     * @throws TableNeedsInnoDbException
+     * @throws Exception
+     */
     public static function createTable(): bool
     {
         $wp = self::getWpPrefix();
         self::checkTableEngineInnoDB("{$wp}terms");
 
         $name = self::getTableName();
+
+        $attributeForeignKey = static::getValidForeignFieldKey("{$name}_storekeeper_attribute_id_fk", $name);
+        $termsForeignKey = static::getValidForeignFieldKey("{$name}_term_id_fk", $name);
```

Comment:
```suggestion
        $attributeForeignKey = static::getValidForeignFieldKey("storekeeper_attribute_id_fk", $name);
        $termsForeignKey = static::getValidForeignFieldKey("term_id_fk", $name);
```
Those would be more clear parameters for me.

Reply:
Changes implemented.